### PR TITLE
Feat/sp 338

### PR DESCRIPTION
**Pull request by muradalamv1**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/divorce/ciccase/model/ApplicantCICDetails.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonNaming(PropertyNamingStrategies.UpperCamelCaseStrategy.class)
+@Builder
+public class ApplicantCICDetails {
+
+
+    @CCD(label = "Is represented by a solicitor?")
+    private YesOrNo solicitorRepresented;
+    @CCD(
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/divorce/ciccase/model/ApplicantCICDetails.java:30]: Minor quibble, but should we begin a flag name with "is" viz. isSolicitorRepresented ?

File: src/main/java/uk/gov/hmcts/divorce/ciccase/model/ApplicantCICDetails.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
+    )
+    private ContactPreferenceType contactDetailsPrefrence;
+
+    @CCD(
+        label = "Email address",
+        typeOverride = Email
+    )
+    private String email;
+
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/divorce/ciccase/model/ApplicantCICDetails.java:63]: Very minor - call it emailAddress, as per the label?

File: src/main/java/uk/gov/hmcts/divorce/ciccase/model/RepresentativeCICDetails.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        label = "What is their contact preference?",
+        access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
+    )
+    private ContactPreferenceType contactDetailsPrefrence;
+    @CCD(
+        label = "Email address",
+        typeOverride = Email
+    )
+    private String email;
+
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/divorce/ciccase/model/RepresentativeCICDetails.java:55]: As above, emailAddress?

**Review by osheaf01 — APPROVED** [review]
Couple of very minor style points, but LGTM

File: src/main/java/uk/gov/hmcts/divorce/ciccase/model/ApplicantCICDetails.java (hunk 1 above)
Comment (spotstory) [reply src/main/java/uk/gov/hmcts/divorce/ciccase/model/ApplicantCICDetails.java:30]: agree

File: src/main/java/uk/gov/hmcts/divorce/ciccase/model/ApplicantCICDetails.java (hunk 2 above)
Comment (spotstory) [reply src/main/java/uk/gov/hmcts/divorce/ciccase/model/ApplicantCICDetails.java:63]: agree

**Review by spotstory — APPROVED** [review]

**Review by jaberelfasi — COMMENTED** [review]
Branch is out of date and needs to be rebased onto master and pushed again

File: src/main/java/uk/gov/hmcts/divorce/ciccase/model/ContactPreferencesDetailsForApplicationCIC.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+import uk.gov.hmcts.divorce.ciccase.model.access.DefaultAccess;
+
+import static uk.gov.hmcts.ccd.sdk.type.FieldType.MultiSelectList;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonNaming(PropertyNamingStrategies.UpperCamelCaseStrategy.class)
+@Builder
+public class ContactPreferencesDetailsForApplicationCIC {
```
Comment (sjamihmcts) [inline src/main/java/uk/gov/hmcts/divorce/ciccase/model/ContactPreferencesDetailsForApplicationCIC.java:20]: Please format the file and remove commented code if not necessary.

File: src/main/java/uk/gov/hmcts/divorce/ciccase/model/SubjectCIC.java [hunk 5]
```diff
@@ -8,13 +8,11 @@
...
 @AllArgsConstructor
 public enum SubjectCIC implements HasLabel {
-    @JsonProperty("SubjectCIC")
+    @JsonProperty("Subject")
     SUBJECT("Subject");
 
-
     private final String label;
-
     public boolean isSubject() {
```
Comment (sjamihmcts) [inline src/main/java/uk/gov/hmcts/divorce/ciccase/model/SubjectCIC.java:15]: Is this method isSubject() required?

**Comment by osheaf01** [comment]
Closing as duplicated by another PR

**Pull request closed without merging**